app/mailers/visitor_mailer.rb

```diff
@@ -21,27 +21,49 @@ def booking_confirmation_email(visit, confirmation, token)
     @confirmation = confirmation
     @token = token
 
-    mail(from: sender, reply_to: prison_mailbox_email, to: recipient, subject: "Visit confirmed: your visit for #{Date.parse(@slot.date).strftime('%e %B %Y').gsub(/^ /,'')} has been confirmed")
+    mail(
+      from: sender,
+      reply_to: prison_mailbox_email,
+      to: recipient,
+      subject: "Visit confirmed: your visit for " \
+       "#{Date.parse(@slot.date).strftime('%e %B %Y').gsub(/^ /,'')} "\
+       "has been confirmed"
```

Comment:
It can localise the subject automatically: it's *name of mailer* → *name of method* → *subject*, *viz.*:

```yaml
en:
  visitor_mailer:
    booking_confirmation_email:
      subject: Whatever
```

However, I'm not sure how interpolated values are passed (or if they can be).